### DISCUSSION TITLE
Are there any open source chat user interfaces that integrate with Portkey?

### DISCUSSION THREAD

**Discussion by timmanik** [Portkey-AI/gateway#624] [Q&A] [answered]
Are there any open source chat user interface like [LibreChat](https://github.com/danny-avila/LibreChat) or [Open WebUI](https://github.com/open-webui/open-webui) that offer integration with Portkey?

Many people I work with are very interested in this integration.

**Comment by vrushankportkey** [comment, marked as answer]
Hey Tim! Sorry should have seen this earlier 😅 We've now added our LibreChat integration doc here - https://portkey.ai/docs/integrations/libraries/librechat

And are also exploring doing the same for Open WebUI now thanks to your nudge!

**Reply by timmanik** [reply]
No worries. Glad we connected on Discord. Thanks for the help!
